src/openrct2/object/ObjectList.cpp

```diff
@@ -22,7 +22,7 @@
 
 // 98DA00
 // clang-format off
-int32_t object_entry_group_counts[] = {
+int32_t object_entry_group_counts[EnumValue(ObjectType::Count)] = {
```

Comment:
This will zero-initialize new entries as `Count` increases, and would only error in case this array exceeds `Count`. It's best to leave the brackets empty, and instead use a static_assert to verify the size.

```cpp
static_assert(std::size(object_entry_group_counts) == EnumValue(ObjectType::Count));
```

Reply:
You'll need to include `<array>` or another header that defined `std::size`.